index.html

```diff
@@ -1,61 +1,174 @@
-            <small class="add-product-form__kcal-info"></small>
-          </div>
-
-          <button class="add-product-form__button" type="submit">add</button>
-        </form>
+  <body>
+    <main class="container">
+      <section class="section-box section-box--products">
+        <h2 class="section-box__title">Products</h2>
+        <ul class="product-list"></ul>
       </section>
 
-      <section class="
-              section-box">
-        <h2 class="section-box__title">Today - 06.01.2019</h2>
-      </section>
-    </section>
-  </main>
+      <section class="main-container">
+        <section class="section-box section-box--add-product">
+          <h2 class="section-box__title">Add product</h2>
+
+          <form class="add-product-form">
+            <div class="add-product-form__wrapper">
+              <label class="add-product-form__label" for="product"
+                >Product</label
+              >
+              <select
+                class="add-product-form__dropdown"
+                id="product"
+                name="product"
+              >
+                <option
+                  class="add-product-form__dropdown-item"
+                  disabled
+                  selected
+                  >Choose a product</option
+                >
+              </select>
+            </div>
+
+            <div class="add-product-form__wrapper">
+              <label class="add-product-form__label" for="amount"
+                >Amount(g)</label
+              >
+              <input
+                class="add-product-form__input"
+                id="amount"
+                name="amount"
+                type="number"
+                min="1"
+                step="1"
+                value="100"
+              />
+            </div>
+
+            <div
+              class="add-product-form__wrapper add-product-form__wrapper--result"
+            >
+              <strong class="add-product-form__kcal-outcome"></strong>
+              <small class="add-product-form__kcal-info"></small>
+            </div>
+
+            <button class="add-product-form__button" type="submit">add</button>
+          </form>
+        </section>
 
-  <script type="text/javascript" src="hydrate-product-lists.js"></script>
-  <script type="text/javascript" src="generate-kcal-result.js"></script>
+        <section
+          class="
+              section-box section-box--summary"
+        >
+          <h2 class="section-box__title">Today - 06.01.2019</h2>
 
-</body>
+          <div class="section-box__wrapper">
+            <div class="section-box__wrapper--left">
+              <h3 class="section-box__title--left">Choosen products:</h3>
+              <ul class="product-list product-list--choosen-products">
+                <div class = "product-list__choosen-products--wrapper">
+                <li
+                  class="product-list__item product-list__item--choosen-products"
```

Comment:
Zauważyłem, że w kodzie poniżej bez przerwy używasz modyfikatora `--chosen-products`, jest to dobry znak do rozpoczęcia korzystania z osobnej klasy bazowej. Tzn. zamiast reużywać np. `product-list--chosen-products`, stwórz inną od `product-list` bazę. Pewnie niektóre style będą się powtarzać, ale nie na tym etapie to nie szkodzi.